### ISSUE TITLE
Obtaining replies from tweet

### ISSUE THREAD

**Issue by DenseLance** [twintproject/twint#432]
### Command Ran
```
import twint

# Configure
c = twint.Config()
c.Get_replies = "731770343052972032"

# Run
twint.run.Search(c)
```
### Description of Issue
I am currently trying to obtain the replies from a particular tweet from https://twitter.com/golan/status/731770343052972032 using the code shown above.
There are about 10 replies to the tweet itself.

It shows:
```
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:noData'items_html'
CRITICAL:root:twint.run:Twint:Feed:Tweets_known_error:'items_html'
'items_html' [x] run.Feed
[!] if get this error but you know for sure that more tweets exist, please open an issue and we will investigate it!
```
Is there a way to use the tweet id to obtain the replies and retweet of a tweet (there is no tweet_id variable)? Another way i have thought of would be:

```
import twint

# Configure
c = twint.Config()
c.Tweet_id = "731770343052972032"
c.Get_replies = True

# Run
twint.run.Profile(c)
```
Also, I would like to know how to get the retweets of a particular tweet/retweet through twint. Thanks for your help!

### Environment Details
Windows, on Python 3.7 (It has no problem when it does normal searching)

**Comment by DenseLance** [comment]
I would also have a UnicodeEncode error when obtaining tweets with unicodes like emojis and other languages.
```
CRITICAL:root:twint.output:_output:UnicodeEncodeError
unicode error [x] output._output
```

**Comment by pielco11** [comment]
```python
import twint

# Configure
c = twint.Config()
c.Get_replies = "731770343052972032"

# Run
twint.run.Search(c)
```

Here the concept is wrong. `config.Get_replies` is boolean, not string. `twint.run.Search` searches for Tweets given a rule (which keywords does it have to contain or not, sent by who to who, that mention specific users, for example)

```python
# Configure
c = twint.Config()
c.Tweet_id = "731770343052972032"
c.Get_replies = True

# Run
twint.run.Profile(c)
```
The `config.Tweet_id` you are talking about exists, and it's called `config.Resume`. Anyway, `config.Resume` does not work with `twint.run.Profile` since this function scrapes the user's timeline.

`config.Get_replies` works with `twint.run.Profile` in such case, Twint will scrape the user's timeline and get all the replies he/she wrote.

If you want to search for tweets that are replies, specify `config.Replies = True`. This will filter the tweets found given a specific pattern (from, to, mentions, etc.)

In conclusion, you can't get retweets or replies for a given tweet, at least now as now


About `UnicodeEncodeError`, there are various articles around that explain how to solve it. This error is not related to Twint, instead to it's to your machine.
